src/rewriters/Rewriter.h

```diff
@@ -78,6 +79,12 @@ class DefaultRewriterConfig {
 public:
     virtual bool previsit(PTRef term) { return true; } // should continue visiting
     virtual PTRef rewrite(PTRef term) { return term; } // don't do anything
+    virtual bool treatIteAsVar() const { return true; }
+};
+
+class IteRewriterConfig : public DefaultRewriterConfig {
```

Comment:
It does not look like the special ite handlining is used anywhere (and that's how it should be).
So I think we don't need to add this.

Reply:
Postponed to the next PR